src/AddIns/Uno.UI.Lottie/ts/Uno.UI.Lottie.ts

```diff
@@ -19,7 +21,7 @@ namespace Uno.UI {
 		private static _runningAnimations: { [id: number]: RunningLottieAnimation } = {};
 		private static _numberOfFrames: number;
 
-		public static setAnimationProperties(newProperties: LottieAnimationProperties): string {
+		public static setAnimationProperties(newProperties: LottieAnimationProperties, animationData?: AnimationData): string {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5477430)